public/app/plugins/datasource/cloud-monitoring/dataquery.gen.ts

```diff
@@ -10,7 +10,7 @@
 
 import * as common from '@grafana/schema';
 
-export const DataQueryModelVersion = Object.freeze([0, 0]);
```

Comment:
@spinillos can we get the plugin version available also in the schemas generated in @grafana/schema?

Reply:
Done :)